btk/catalog.py

```diff
@@ -109,3 +117,6 @@ def _prepare_table(self):
         self._raw_catalog["ref_mag"] = self._raw_catalog["MAG"]
         self._raw_catalog["btk_size"] = size
         return table
+
+    def get_galsim_catalog(self):
+        return self.galsim_catalog
```

Comment:
Yes, as @thuiop  mentioned. I think we should keep either raw_catalog or galsim_catalog but not both. I think `_raw_catalog` was a way to standardized the different catalogs. But since we only have two types I don't see it like a super useful attribute right now. I you want to remove it and just keep `galsim_catalog` that is fine.

Reply:
The only time where it would be useful would be if we had a DrawBlendsGenerator working for several different catalogs but I do not see that happening often.